src/components/teams/Teams.js

```diff
@@ -128,21 +128,52 @@ export default class Teams extends Component {
             </thead>
             <tbody>
              {matchesForTeam.map(match => {
-               return (
-                <tr key={match.id} onClick={this._openMatch.bind(this, match.id)} className="clickable">
-                  <td>{match.date.format('DD/MM')}</td>
-                  <td>{match.date.format('HH:mm')}</td>
-                  <td>{match.isHomeMatch ? this._renderOwnTeamTitle(team) : this._renderOpponentTitle(match)} -
-                  {match.isHomeMatch ? ' ' + this._renderOpponentTitle(match) : ' ' + this._renderOwnTeamTitle(team)}</td>
-                  <td>{this._renderScores(match)}</td>
-                </tr>
-              );
+               return this._getMatch(match,team);
              })}
             </tbody>
         </table>
       );
   }
 
+  _getMatch(match,team) {
+    if (this._won(match)) {
+      var styling = {backgroundColor:'PaleGreen'};
+      return (
+       <tr key={match.id} style={styling} onClick={this._openMatch.bind(this, match.id)} className="clickable">
+          <td>{match.date.format('DD/MM')}</td>
+          <td>{match.date.format('HH:mm')}</td>
+          <td>{match.isHomeMatch ? this._renderOwnTeamTitle(team) : this._renderOpponentTitle(match)} -
+             {match.isHomeMatch ? ' ' + this._renderOpponentTitle(match) : ' ' + this._renderOwnTeamTitle(team)}</td>
+          <td>{this._renderScores(match)}</td>
+       </tr>
```

Comment:
Je herhaalt hier nu 2x dezelfde code
Bij elke wijziging hieraan, ga je de wijziging twee keer moeten doen. De helft van de keren betekent dit dat slechts een van de twee plaatsen aangepast wordt en dat de site inconsistent is....

Wat je hier kan doen is:

```
return (<tr className={cn('match-success': this._won(match))} ... />);
```

Dus ook ipv PaleGreen, werk met classes (Match.css):
```
.match-lost {
  background-color: #FF6666;
}

.match-won {
  background-color: #f0ad4e;
}

.match-draw {
  background-color: #A2C93A;
}
```

Check het package classNames: https://github.com/JedWatson/classnames